src/tests/useQuery.test.js

```diff
@@ -506,4 +506,24 @@ describe('useQuery', () => {
     rendered.getByText('data')
     expect(rendered.getByTestId('isFetching').textContent).toBe('false')
   })
+
+  it('should initialize state properly, when initialData is falsy', async () => {
```

Comment:
```suggestion
  it('should initialize state properly, when initialData is undefined', async () => {
```

Reply:
this test checks falsy, but defined initialData

Reply:
Got it